eventmesh-security-plugin/eventmesh-security-api/src/test/java/org/apache/eventmesh/api/common/ConfigurationWrapperTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.api.common;
+
+import java.io.File;
+import java.net.URL;
+import java.util.Properties;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ConfigurationWrapperTest {
+
+    @Test
+    public void testGetDefaultConfig() {
+        try {
+            URL resource = Thread.currentThread().getContextClassLoader().getResource("test.properties");
+            String filePath = resource.getPath();
+            File f = new File(filePath);
+            String path = f.getParent();
+            System.setProperty("confPath", path + File.separator + "1");
```

Comment:
maybe we can use a more elegant test property file path instead of `/resources/1` here?

Reply:
Thanks!
I refactor it.

Reply:
@horoc done.